display/ssd1306.h

```diff
@@ -280,6 +292,36 @@ class SSD1306 : public I2CDevice, Looper, StateMachine, SaberBase, private Audio
     }
   }
 
+  void SB_On() override {
+    if (IMG_on) {
```

Comment:
might be easier if we put this test inside ShowFile

Reply:
done.